Wrong Way Cow puzzle

dencee opened the pull request:
2D array puzzle from:
https://www.codewars.com/kata/the-wrong-way-cow

This puzzle may be a bit too difficult (5 kyu), but I thought it was fun to solve. I'll leave it up to you if you think it's too difficult for level 4. The WrongWayCowTest.java file is run to check if the proposed solution is correct.

dencee committed:
Wrong Way Cow puzzle

2D array puzzle from:
https://www.codewars.com/kata/the-wrong-way-cow

keithagroves commented:
Thanks Daniel! I have a few  request:  

- move the two classes into another package called `extras.the_wrong_way_cow` 
- The method should `return null` so there are no errors in the code.
- add `<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/4"/>` to the classpath so that junit is used (this should be done automatically when you import junit in eclipse).

Let me know if you need any help!

dencee committed:
Made changes from pull request

1. moved to extras.the_wrong_way_cow package
2. findWrongWayCow returns null so it compiles.
3. added Junit to the classpath
